### ISSUE TITLE
Bash Completion not installed in deb package

### ISSUE THREAD

**Issue by jdrovl** [NVIDIA/enroot#164]
There is a bash completion script at `enroot/conf/bash_completion` but it does not get installed to `/usr/share/bash_completion/completions/enroot` by the `.deb` installation as expected. I can't see any reference to it in the install script (or the manual install instructions), maybe it should be added since it's quite handy (as in Docker) to be able to tab-complete image names etc.

**Comment by 3XX0** [comment]
It should be installed in `/usr/share/enroot` as an extra config. You can just link/copy the file as necessary.

**Comment by jdrovl** [comment]
Fair enough, I added it with `sudo ln /usr/share/enroot/bash_completion  /usr/share/bash-completion/completions/enroot`. I just filed an issue because I expected the installer to do this.
